attributes/volume_attachments_variables.rb

```diff
@@ -23,16 +23,16 @@
 }
 
 # use this to create server profile or server profile template with volume attachments using api500 or greater
-volume_data_api_500 = {
-  name: 'Volume1',
-  description: 'Volume store serv for api500',
-  size: 1024 * 1024 * 1024,
-  provisioningType: 'Thin',
-  isShareable: false
-}
+# volume_data = {
```

Comment:
commented?

Reply:
to make it easier for clients to run the example with api500 just by uncommenting this code